### ISSUE TITLE
Missing additional download

### ISSUE THREAD

**Issue by inobrevi** [microsoft/Bringing-Old-Photos-Back-to-Life#248]
```
# pull the syncBN repo
%cd photo_restoration/Face_Enhancement/models/networks
!git clone https://github.com/vacancy/Synchronized-BatchNorm-PyTorch
!cp -rf Synchronized-BatchNorm-PyTorch/sync_batchnorm .
%cd ../../../

%cd Global/detection_models
!git clone https://github.com/vacancy/Synchronized-BatchNorm-PyTorch
!cp -rf Synchronized-BatchNorm-PyTorch/sync_batchnorm .
%cd ../../

# download the landmark detection model
%cd Face_Detection/
!wget http://dlib.net/files/shape_predictor_68_face_landmarks.dat.bz2
!bzip2 -d shape_predictor_68_face_landmarks.dat.bz2
%cd ../

# download the pretrained model
%cd Face_Enhancement/
!wget https://facevc.blob.core.windows.net/zhanbo/old_photo/pretrain/Face_Enhancement/checkpoints.zip
!unzip checkpoints.zip
%cd ../

%cd Global/
!wget https://facevc.blob.core.windows.net/zhanbo/old_photo/pretrain/Global/checkpoints.zip
!unzip checkpoints.zip
%cd ../
```

```
/content/photo_restoration
/content/photo_restoration/Face_Enhancement
--2022-11-09 21:01:48--  https://facevc.blob.core.windows.net/zhanbo/old_photo/pretrain/Face_Enhancement/checkpoints.zip
Resolving facevc.blob.core.windows.net (facevc.blob.core.windows.net)... 20.60.228.1
Connecting to facevc.blob.core.windows.net (facevc.blob.core.windows.net)|20.60.228.1|:443... connected.
HTTP request sent, awaiting response... 404 The specified resource does not exist.
2022-11-09 21:01:49 ERROR 404: The specified resource does not exist..

unzip:  cannot find or open checkpoints.zip, checkpoints.zip.zip or checkpoints.zip.ZIP.
/content/photo_restoration
/content/photo_restoration/Global
--2022-11-09 21:01:49--  https://facevc.blob.core.windows.net/zhanbo/old_photo/pretrain/Global/checkpoints.zip
Resolving facevc.blob.core.windows.net (facevc.blob.core.windows.net)... 20.60.228.1
Connecting to facevc.blob.core.windows.net (facevc.blob.core.windows.net)|20.60.228.1|:443... connected.
HTTP request sent, awaiting response... 404 The specified resource does not exist.
2022-11-09 21:01:50 ERROR 404: The specified resource does not exist..

unzip:  cannot find or open checkpoints.zip, checkpoints.zip.zip or checkpoints.zip.ZIP.
/content/photo_restoration
```

**Comment by seppemarotta** [comment]
Same Issue here, I was unable to download the checkpoints.zip

**Comment by inobrevi** [comment]
I figured it out, you need ti download release and manually extract the checkpoints into right place.

**Comment by seppemarotta** [comment]
> I figured it out, you need ti download release and manually extract the checkpoints into right place.

I've just done that but I always get `_You are using the mapping model without global restoration._` , did you manage to make Global Restoration work ? @adamsatyr

**Comment by inobrevi** [comment]
I managed to make it fully functional, at least in Google collab.

Probably you didn't put global check point in correct folder or skipped extraction of syncBN repo

**Comment by jolzi** [comment]
Any infos on this? Manually downloaded and extracted checkpoints to Global/checkpoints and Face_Enhancement/checkpoints. 
Cloned and copied syncBN repo to final paths Face_Enhancement/models/networks/sync_batchnorm and Global/detection_models/sync_batchnorm.
Is this correct? I still don't get global restoration

**Comment by ZRomene** [comment]
`%cd photo_restoration/Face_Enhancement/models/networks
!git clone https://github.com/vacancy/Synchronized-BatchNorm-PyTorch
!cp -rf Synchronized-BatchNorm-PyTorch/sync_batchnorm .
%cd ../../../

%cd Global/detection_models
!git clone https://github.com/vacancy/Synchronized-BatchNorm-PyTorch
!cp -rf Synchronized-BatchNorm-PyTorch/sync_batchnorm .
%cd ../../

%cd Face_Detection/
!wget http://dlib.net/files/shape_predictor_68_face_landmarks.dat.bz2
!bzip2 -d shape_predictor_68_face_landmarks.dat.bz2
%cd ../

%cd Face_Enhancement/
!wget https://github.com/microsoft/Bringing-Old-Photos-Back-to-Life/releases/download/v1.0/face_checkpoints.zip
!unzip face_checkpoints.zip
%cd ../

%cd Global/
!wget https://github.com/microsoft/Bringing-Old-Photos-Back-to-Life/releases/download/v1.0/global_checkpoints.zip
!unzip global_checkpoints.zip
%cd ../`

You must correct the paths to extract the checkpoints.  
Use `!wget https://github.com/microsoft/Bringing-Old-Photos-Back-to-Life/releases/download/v1.0/face_checkpoints.zip
!unzip face_checkpoints.zip `  in the place of `!wget https://facevc.blob.core.windows.net/zhanbo/old_photo/pretrain/Face_Enhancement/checkpoints.zip
!unzip checkpoints.zip `

And `!wget https://github.com/microsoft/Bringing-Old-Photos-Back-to-Life/releases/download/v1.0/global_checkpoints.zip
!unzip global_checkpoints.zip` in the place of `!wget https://facevc.blob.core.windows.net/zhanbo/old_photo/pretrain/Global/checkpoints.zip
!unzip checkpoints.zip`

Fully worked for me !

**Comment by spatiallysaying** [comment]
Better to use full path. Facing issues related path with previous ways.
```

%cd /content/photo_restoration/Face_Enhancement/models/networks
!git clone https://github.com/vacancy/Synchronized-BatchNorm-PyTorch
!cp -rf Synchronized-BatchNorm-PyTorch/sync_batchnorm .
%cd ../../../

%cd /content/photo_restoration/Global/detection_models
!git clone https://github.com/vacancy/Synchronized-BatchNorm-PyTorch
!cp -rf Synchronized-BatchNorm-PyTorch/sync_batchnorm .
%cd ../../

%cd /content/photo_restoration/Face_Detection/
!wget http://dlib.net/files/shape_predictor_68_face_landmarks.dat.bz2
!bzip2 -d shape_predictor_68_face_landmarks.dat.bz2
%cd ../

%cd /content/photo_restoration/Face_Enhancement/
!wget https://github.com/microsoft/Bringing-Old-Photos-Back-to-Life/releases/download/v1.0/face_checkpoints.zip
!unzip face_checkpoints.zip
%cd ../

%cd /content/photo_restoration/Global/
!wget https://github.com/microsoft/Bringing-Old-Photos-Back-to-Life/releases/download/v1.0/global_checkpoints.zip
!unzip global_checkpoints.zip
%cd ../
```
